app/views/step_by_step_pages/show/_actions.erb

```diff
@@ -29,20 +29,19 @@
       <% end %>
 
       <% if @step_by_step_page.can_discard_changes? %>
-        <%= button_to 'Discard changes', { action: "revert", step_by_step_page_id: @step_by_step_page.id }, data: { confirm: 'This will delete your draft. Do you want to discard your changes?' }, class: "gem-c-button govuk-button govuk-button--warning" %>
+        <%= link_to 'Discard changes', { action: "revert", step_by_step_page_id: @step_by_step_page.id }, method: :post, data: { confirm: 'This will delete your draft. Do you want to discard your changes?' }, class: %w(govuk-link govuk-link--no-visited-state app-link--destructive) %>
       <% end %>
     <% end %>
 
     <% if @step_by_step_page.can_be_unpublished? || @step_by_step_page.can_be_deleted? %>
       <% if @step_by_step_page.can_be_unpublished? %>
-        <%= render "govuk_publishing_components/components/button", {
-          text: "Unpublish",
-          href: step_by_step_page_unpublish_path(@step_by_step_page),
-          destructive: true
-        } %>
+        <%= link_to "Unpublish",
+          step_by_step_page_unpublish_path(@step_by_step_page),
+          class: %w(govuk-link govuk-link--no-visited-state app-link--right)
+        %>
       <% end %>
       <% if @step_by_step_page.can_be_deleted? %>
-        <%= button_to "Delete", @step_by_step_page, method: :delete, data: { confirm: 'Deleted step by steps cannot be recovered. Are you sure you want to delete this step by step page?' }, class: "gem-c-button govuk-button govuk-button--warning" %>
+        <%= link_to "Delete", @step_by_step_page, method: 'delete', data: { confirm: 'Deleted step by steps cannot be recovered. Are you sure you want to delete this step by step page?' }, class: %w(govuk-link govuk-link--no-visited-state app-link--destructive) %>
```

Comment:
Because the element changed from a button to a link. This seemed to me to be the way to do it judging by other examples in the app, but I may be wrong.
- https://github.com/alphagov/collections-publisher/blob/3ab554382531a31f35b2521dda06f08bbabceba8/app/views/step_by_step_pages/show.html.erb#L60
- https://github.com/alphagov/collections-publisher/blob/master/app/views/secondary_content_links/index.html.erb#L107

Reply:
I think either should work.

Reply:
Will test and use the symbol if that's the case.